master/custom/workers.py

```diff
@@ -281,4 +281,10 @@ def get_workers(settings):
             parallel_tests=2,
             parallel_builders=2,
         ),
+        cpw(
+            name="ambv-bb-win11",
+            tags=['windows', 'win11', 'amd64', 'x86-64', 'bigmem'],
+            branches=['3.9', '3.10','3.11', '3.x'],
+            parallel_tests=8,
```

Comment:
How is this used?

Reply:
This should go to the regrtest `-j` option to set the number of parallel test workers explicitly.